src/app/dynamic-forms/models/cinchy-form.model.ts

```diff
@@ -826,7 +826,7 @@ export class Form {
         null
       );
 
-      return query;
+      return this.childFormLinkId != null ? query : null;
```

Comment:
There's no reason to do the work of building the query if we're just going to discard it based on a boolean condition. Just add `if (!this.childFormLinkId)` on line 815 and return null if it's truthy

Reply:
Hi @DAquilinaCinchy , I have resolved above comment.